ui/screens.py

```diff
@@ -1096,6 +1104,11 @@ def stacked(self):
 
     def stackAccounts(self):
         self.app.appWindow.stack(self.accountScreen)
+    def stackAgendas(self):
+        if len(self.agendasScreen.agendas) == 0 or self.accountScreen.currentPoolVoteBits < 0:
+            self.app.appWindow.showError("pool not yet synced")
```

Comment:
TD did not sync my vote choices when I connected my first pool, causing this message to be shown when clicking the "Voting" button. I had to restart.